packages/dx-scheduler-core/src/utils.ts

```diff
@@ -121,12 +151,14 @@ export const adjustAppointments = (groups, byDay = false) => groups.map((items)
   return { items: appointments, reduceValue };
 });
 
-export const calculateFirstDateOfWeek = (currentDate, firstDayOfWeek, excludedDays = []) => {
+export const calculateFirstDateOfWeek: CalculateFirstDateOfWeekFn = (
+  currentDate, firstDayOfWeek, excludedDays = [],
+) => {
   const currentLocale = moment.locale();
   moment.updateLocale('tmp-locale', {
-    week: { dow: firstDayOfWeek },
+    week: { dow: firstDayOfWeek, doy: 1 }, // doy
```

Comment:
// doy

Is it a required comment?

Reply:
@SergeyAlexeev,

I will change this comment. The property `doy` is required for TS using